testing/test_hsm.py

```diff
@@ -1143,9 +1127,11 @@ def test_boot_to_hsm_unlock(start_hsm, hsm_status, enter_local_code):
     enter_local_code('123123')
     time.sleep(.5)
     assert hsm_status().active == False
-    assert hsm_status().policy_available == False
+    assert hsm_status().policy_available == True  # we haven't removed anythong why shoudl it be not available?
```

Comment:
Not sure why this was False before, but was not working --> maybe I'm missing something here ?

Reply:
I think an earlier version of the firmware always deleted the HSM policy file when unlocked in this manner, but then that got removed (since hard to reload policy) and so this test became wrong.